.mocharc.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
+    extension: 'js',
+    forbidOnly: true,
+    noInsight: true,
```

Comment:
Does not work.

Reply:
Interesting, thanks! I'll look into why it doesn't and put it back like before

Reply:
Just ask.
https://github.com/jhipster/generator-jhipster/blob/6b3cbfa43e5ee8a4faacc9fc624b2d4836c8c1a5/generators/statistics.js#L47

Reply:
awesome :) thanks